### DISCUSSION TITLE
"Hide entered" vs "hide completed"

### DISCUSSION THREAD

**Discussion by Lasalle42** [iSoron/uhabits#1832] [Help & FAQ]
The change from hide completed to hide entered makes it so that I can't make multiple entries to a numerically measurable goal before completing it. 

For example: I have a measurable goal to walk 10,000 steps a day. If I make an entry of 2,500 steps, it hides the task even though I haven't reached my goal yet. 

Same thing with my 8 cups of water task. I track them throughout the day.

Its frustrating now that I have to keep toggling and untangling the "hide entered" to make updates to those measurable goals.

**Comment by LCStark** [comment]
Yeah, I'm having the same issues few years later. Currently when question marks are disabled you get two filter options - "hide archived" and "hide completed". Enabling question marks replaces the "hide completed" filter with "hide entered". That is a problem, because I want to be able to see days with no data entered, but that breaks my workflow with measurable tasks.

I have tasks that I'm doing several times each day, and since you can't set frequency higher that 1/day with "Yes or No" tasks, I'm using "Measurable" ones for that, setting the amount of daily repetitions and adding each one after it's done. The problem is, when "Show question marks for missing data" is enabled, instead of hiding the habit when it has been completed for the day, LHT hides it after first entry, meaning that for each repetition on that day I have to disable the filter first, then change the amount, then re-enable it to keep my current list clean.

**Comment by Kristina-Brankov** [comment]
Hi! I would really appreciate a “Hide manually entered today” option that ignores automatic/implicit checkmarks.

For example, if my goal is language learning 3× per week, after reaching the weekly goal I’d still like the habit to appear on subsequent days as optional, and only be hidden after I actually check it that day.

This would make managing a longer habit list much easier. Thank you!
